File: DSA/Prarthana_Baisoya_590024838/Day1/q2.c
#include <stdio.h>

int findSum(int arr[], int size) {
    int sum = 0;

    for(int i = 0; i < size; i++) {
        sum += arr[i];
    }

    return sum;
}

int main() {
    int arr[] = {3, 8, 2, 9, 1};
    int size = sizeof(arr) / sizeof(arr[0]);

    printf("Sum of array elements: %d\n", findSum(arr, size));

    return 0;
}

File: DSA/Madhav_590014266/Day1/Q2.java
public class SumArray {
    public static int sumArray(int[] arr) {
        int sum = 0;

        for (int num : arr) {
            sum += num;
        }

        return sum;
    }

    public static void main(String[] args) {
        int[] arr = {3, 8, 2, 9, 1};

        int result = sumArray(arr);
        System.out.println("Sum = " + result);
    }
}